onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -280,11 +280,20 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformersForRuntimeO
   switch (level) {
     case TransformerLevel::Level1:
       break;
-    case TransformerLevel::Level2:
+    case TransformerLevel::Level2: {
+      const InlinedHashSet<std::string_view> cpu_cuda_rocm_acl_armnn_eps = {onnxruntime::kCpuExecutionProvider,
+                                                                            onnxruntime::kCudaExecutionProvider,
+                                                                            onnxruntime::kRocmExecutionProvider,
+                                                                            onnxruntime::kAclExecutionProvider,
+                                                                            onnxruntime::kArmNNExecutionProvider};
+
+      transformers.emplace_back(std::make_unique<ConvActivationFusion>(cpu_cuda_rocm_acl_armnn_eps,
+                                                                       apply_context));
```

Comment:
Should we keep the list of EPs minimal given we only expect ORT format models to be used with the CPU EP currently?

Also should we keep the same ordering as a full build - namely the QDQ optimizer is first followed by others? Currently I don't think it matters, but it might for future additions so everything being consistent will make it easier to do the right thing later.